src/components/Teams.js

```diff
@@ -0,0 +1,47 @@
+// Teams.js
```

Comment:
very welled organized code. But if you would add some comment. so it would definitely help reviewer. 👍🏻  👏🏻

Reply:
YES I am so sorry, I never considered anyone else would be reading this code!! I have absolutely learned for next time

Reply:
Thanks mate!